x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -331,8 +336,8 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
                 </EuiFlexItem>
                 <EuiFlexItem grow={false}>
                   {timelinesUi.getLastUpdated({
-                    updatedAt: updatedAt || 0,
-                    showUpdating: loading,
+                    updatedAt: updatedAt || Date.now(),
```

Comment:
Tested, and looks good. I wonder if we should do something here like this so it doesn't even show till we have the updatedAt?

```
{updatedAt && timelinesUi.getLastUpdated({
  updatedAt,
  showUpdating,
})}
```

It ends up looking like:


https://user-images.githubusercontent.com/17211684/132040083-8281a3b4-2a7b-4ca2-a6b9-8b45b253bf9a.mp4

Reply:
yeah we could go with that instead, cover that little gap in the little